angr/procedures/posix/getenv.py

```diff
@@ -0,0 +1,36 @@
+import angr
+
+######################################
+# getenv
+######################################
+
+class getenv(angr.SimProcedure):
+    '''
+    The getenv() function searches the environment list to find the
+    environment variable name, and returns a pointer to the
+    corresponding value string.
+    '''
+    #pylint:disable=arguments-differ
+
+    def run(self, m_name):
+        strlen = angr.SIM_PROCEDURES['libc']['strlen']
+        name_len = self.inline_call(strlen, m_name)
+        name_expr = self.state.memory.load(m_name, name_len.max_null_index, endness='Iend_BE')
+        name = self.state.solver.eval(name_expr, cast_to=bytes)
+
+        p = self.state.posix.environ
+        if p is None:
+            return self.state.solver.BVS(b"getenv__" + name, self.state.arch.bits)
```

Comment:
can you add the `key` argument to this BVS call so that it shows up in state.solver.describe_variables and co?